src/variables/integration/dropdown.spec.js

```diff
@@ -0,0 +1,20 @@
+describe("test dropdown menu", () => {
+  it("should open dropdown menu", () => {
```

Comment:
test it under one `it`(reduce all your test cases into one) - `it("should open dropdown and pick a value")`